frontend/src/pages/ParentsPage.js

```diff
@@ -28,6 +29,12 @@ class ParentsPage extends Component {
       name: 'Results',
       component: Results
     }
+    // ,
+    // {
+    //   route: '/parents/advice',
+    //   name: 'Advice',
+    //   component: Advice //should i be using the ParentsAdvice page or should i just delete that page completely
```

Comment:
Let's remove the navigation code and have the `ParentsPage` JUST render advice. So move the code (just be sure to check you're not replacing anything important haha) from `ParentAdvice` to `ParentsPage`. And then we can remove the stuff that you added for navigating to `ParentAdvice`. Sorry for the change -- our vision of the parents page changed after speaking w Whitney.